controllers/base_controller.js

```diff
@@ -117,6 +117,9 @@ module.exports = function BaseControllerModule(pb) {
         this.body                = props.body;
         this.localizationService = props.localization_service;
         this.ls                  = this.localizationService;
+        this.ls.activeTheme      = props.activeTheme;
+        this.ls.siteName         = props.siteName;
+        this.ls.site             = props.site;
```

Comment:
Seems redundant to set these here.  Why not pass them in as options when the localization instance is created in the RequestHandler?